ArcGISRuntimeSDKQt_QMLSamples/Features/FeatureLayer_Query/FeatureLayer_Query.qml

```diff
@@ -127,15 +96,18 @@ Rectangle {
                 selectByMouse: true
                 validator: RegExpValidator{ regExp: /^[a-zA-Z ]*$/ }
                 Keys.onReturnPressed: {
-                    query();
+                    btn.clicked();
                 }
             }
 
             Button {
+                id: btn
                 text: "Find and Select"
                 enabled: featureTable.loadStatus === Enums.LoadStatusLoaded
                 onClicked: {
-                    query();
+                    queryFeatures(findText.text, featureTable) // query the table
```

Comment:
I did it this way because the function would be more encapsulated. You could theoretically pass in any search string and feature table and use this pattern. But I now see some slight refactoring I need to do to move the SQL clause elsewhere if I'm going to do that